app/views/contexts/thank_you_context.py

```diff
@@ -75,12 +78,12 @@ def build_view_submitted_response_context(
 
 
 def build_census_thank_you_context(
-    metadata: MetadataProxy,
+    metadata: MetadataProxy | dict,
     confirmation_email_form: Optional[EmailForm],
     form_type: str,
 ) -> dict[str, Union[bool, str, None]]:
     context = {
-        "display_address": metadata["display_address"],
+        "display_address": metadata["display_address"],  # type: ignore
```

Comment:
Missing type ignore explanation

Reply:
I'm not sure why this is here as there is no error when i removed it. Will change